io-ballerina/build.gradle

```diff
@@ -177,6 +180,9 @@ task ballerinaBuild {
 
 task ballerinaPublish {
     doLast {
+        if (project.version.endsWith('-SNAPSHOT')) {
+            return
+        }
```

Comment:
Shall we add the exception when the token is not available?

Reply:
Added in here https://github.com/ballerina-platform/module-ballerina-io/pull/88/commits/6f0645918f8883296f0941f6d4cf3641eb8fa31e